src/module.ts

```diff
@@ -1656,7 +1655,7 @@ export class Module {
           passes.push("simplify-locals");
           passes.push("vacuum");
 
-          passes.push("inlining-optimizing");
+          passes.push("inlining");
```

Comment:
Just curious: What's the different between `inlining` and `inlining-optimizing`? Is the latter unnecessary?

Reply:
It seems `inlining-optimizing` also apply constant propagation but only for specific part or something like this